### DISCUSSION TITLE
Arc light UI does not show new MWL AET

### DISCUSSION THREAD

**Discussion by fonteneu** [dcm4che/dcm4chee-arc-light#4610] [Q&A] [answered]
A new AE has been created in the UI2 with MWL and MPPS capabilities. The default WORKLIST has been deleted via the UI.

Now go to Navigation and MWL. In the AE dropdown, only WORKLIST is shown despite being deleted.

The new AE "MODMWL" does not appear. This is the "Called AET" that modalities use to query worklist and do MPPS.

Seems like a bug? If not, how can I set the Called (not calling) AET for the MWL and MPPS SCP's ?

It's the called AET in the sense of MODMWL in `mppsscu -c MODMWL@myhost:11112`

**Comment by vrindanayak** [comment, marked as answer]
> Now go to Navigation and MWL. In the AE dropdown, only WORKLIST is shown despite being deleted.
> Seems like a bug? If not, how can I set the Called (not calling) AET for the MWL and MPPS SCP's ?

What you see in Navigation - MWL in UI is not AE dropdown, but Web Applications dropdown. Seems you forgot to create a Web Application for the new AE you created - see https://github.com/dcm4che/dcm4chee-arc-light/wiki/AE-Title%28s%29-of-the-Archive#ae-for-web-services

**Reply by fonteneu** [reply]
OK. I will add that as well. I find Arc Light to be _significantly_ harder to configure for our use case than 2.x. Especially the ldap madness.

Therefore I appreciate your help very much.

**Reply by fonteneu** [reply]
@vrindanayak but even if I add those, the worklist SCP only accept "WORKLIST" as calling AET, not my "MODMWL". I have added correct presentation context and transfer syntaxes on MODMWL.

Are there more places I need to configure my new AE?
